### DISCUSSION TITLE
Can't connect to PostgreSQL

### DISCUSSION THREAD

**Discussion by chucky23** [Budibase/budibase#2834] [Help] [answered]
I created a database in Heroku, but Badibase doesn't want to connect to it. Appsmith connects without problems with the same settings.

SSL off: https://drive.google.com/uc?id=1N0l9kt9d07VFDIY34XC_gON5D6j7jS5Z
SSL on: https://drive.google.com/uc?id=14tyBk1McA0V4-3BpVjUJN0ooyyUku2Oh

**Comment by mjashanks** [comment, marked as answer]
Hey @chucky23 - thanks for reporting. We have an issue logged for this, and intend to release a fix for this at the end of October - https://github.com/Budibase/budibase/issues/1967
